kratos/python/add_points_to_python.cpp

```diff
@@ -68,7 +68,7 @@ void  AddPointsToPython(pybind11::module& m)
     
     class_<Point, Point::Pointer, array_1d<double,3> >(m,"Point") //WARNING: this was previously called Point3D
     .def(init<double, double, double>())
-    .def(init<vector_expression<vector<double> > >())
+    .def(init<vector_expression<Vector> >())
```

Comment:
shouldn't this be DenseVector ?

Reply:
DenseVector<double> is equivalent to Vector. Both options are valid.